### DISCUSSION TITLE
Error with Google authentication

### DISCUSSION THREAD

**Discussion by sidtechno** [brocoders/extensive-react-boilerplate#413] [Q&A] [answered]
Hi all,

i'm new to react and that boilerplate. when i change the NEXT_PUBLIC_GOOGLE_CLIENT_ID value to my Google credential key, i receive an error 500

![image](https://github.com/brocoders/extensive-react-boilerplate/assets/10188381/12303e05-c5fd-461d-a414-1dbca46fb24f)

in my google credential, i added https://localhost and https://localhost:3000 in javascript origins, but the problem seem to be with the callback url : 

/ 20240628145538
// https://nestjs-boilerplate-test.herokuapp.com/api

{
  "message": "Cannot GET /api",
  "error": "Not Found",
  "statusCode": 404
}

Not sure what i'm missing.

**Comment by Shchepotin** [comment, marked as answer]
@sidtechno you need to run your own backend based on this
